@here/harp-examples/src/getting-started_globe-projection.ts

```diff
@@ -18,6 +18,7 @@ export namespace GlobeExample {
         const mapView = new MapView({
             canvas,
             theme: "resources/berlin_tilezen_base_globe.json",
+            maxVisibleDataSourceTiles: 360,
```

Comment:
Please remove. We should not hide the fact that we are showing to many tiles close to the poles. There is a ticket for it as well and we should address it properly.

Reply:
Done. Thanks.